boards/DIM/Src/App/App_DimWorld.c

```diff
@@ -76,6 +78,23 @@ void App_DimWorld_Destroy(struct DimWorld *world)
     free(world);
 }
 
+uint32_t App_DimWorld_GetCurrentSecond(struct DimWorld *world)
+{
+    return world->current_ms / 1000;
+}
+
+uint32_t App_DimWorld_GetCurrentMillisecond(struct DimWorld *world)
+{
+    return world->current_ms;
+}
+
+void App_DimWorld_SetCurrentMillisecond(
```

Comment:
Similar thought with naming these functions, we might want to say `SetCurrentTimeMilliseconds` just to be super clear (and similarly for the other ones).

Reply:
Fixed! Went with `SetCurrentTimeInMilliseconds` (Same as yours, except with an extra `In`)